public/arethusa/dist/arethusa.dep_tree.concat.js

```diff
@@ -150,7 +150,7 @@ angular.module('arethusa.depTree').service('depTree', [
 
     this.externalDependencies = {
       ordered: [
-        "/vendor/d3.js",
+        "https://cdnjs.cloudflare.com/ajax/libs/d3/3.4.13/d3.js",
```

Comment:
I'd like to keep the Arethusa code in this repository as up-to-date with the real Arethusa code as possible. But at the same time I'd like to keep this repository from depending on external services as much as possible (https://github.com/perseids-publications/treebank-template/commit/79fa03c13f6c54e533f4a1ec416d4d0afdcab963#diff-d8af7a7319d6fd556f18f0be43c7d603).